contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -39,11 +39,14 @@ contract TrueCreditAgency is UpgradeableClaimable {
 
     mapping(ITrueFiPool2 => mapping(address => uint8)) public creditScore;
     mapping(ITrueFiPool2 => mapping(address => uint256)) public borrowed;
+    mapping(ITrueFiPool2 => mapping(address => uint256)) public nextRepaymentTerm;
```

Comment:
Not sure if the word `term` fits well here. Maybe `date` or `timestamp`? Not sure tbh.